manifests/init.pp

```diff
@@ -222,6 +559,150 @@
     }
   }
 
+  if type($enable_db_audit_user) == 'boolean' {
```

Comment:
`type` is now a reserved word. Instead use `is_string()` https://github.com/puppetlabs/puppetlabs-stdlib/tree/3.2.0#is_string